src/styles/theme.css

```diff
@@ -117,9 +117,9 @@
   --amino-shadow-button-warning: 0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset,
     0px 1px 3px 0px rgba(229, 89, 26, 0.4);
   --amino-button-focus-ring: 0 0 0 2px var(--amino-blue-600);
-  --amino-shadow-button-standard: 0px 1px 3px 0px rgba(110, 110, 120, 0.12),
-    0px -1px 1px 1px rgba(0, 0, 0, 0.08) inset,
-    0px 0px 0px 1px rgba(106, 109, 119, 0.12) inset;
+  --amino-shadow-button-standard: 0px 1px 3px 0px rgba(110, 110, 120, 0.12), 
+    0px 0px 0px 1px rgba(106, 109, 119, 0.12) inset, 
+    0px -1px 1px 1px rgba(0, 0, 0, 0.08) inset;
```

Comment:
This theme.css is generated. Changes that are made manually will be disregarded and have no effect at all.
We are still improving our docs. Can you check this `README` out, it's about building theme process. https://github.com/Zonos/amino/tree/main/build-utils/css#building-process

Please let me know if you have any questions, I can pair with you on this if you want 😉

Reply:
Ah ok. I saw there was a process for updating those files but wasn't sure where it was. Ill update those files and then Ill ask you to come over if/when I need help running the script.

Reply:
Sounds good 😉 
Also I see in the PR description, you said you want to "update to shadows on dark mode buttons", does that mean you want to update box shadow on the button on dark mode or you want to apply box shadow of dark mode button to the light mode button?